src/Template/Element/sentences/list_form.ctp

```diff
@@ -0,0 +1,43 @@
+<?php
+use App\Model\CurrentUser;
+use Cake\ORM\TableRegistry;
+
+$SentencesLists = TableRegistry::getTableLocator()->get('SentencesLists');
+$lists = $SentencesLists->getUserChoices(
+    CurrentUser::get('id'), $sentenceId, true
+);
+$mostRecentList = $this->request->getSession()->read('most_recent_list');
+$listsJSON = htmlspecialchars(json_encode($lists), ENT_QUOTES, 'UTF-8');
+?>
+
+<form layout="column" style="border-top: 1px solid #f1f1f1; padding-top: 10px;"
```

Comment:
Any reason for not putting this style (and the other ones later in that file) into `elements.css`?

Reply:
No reason other than laziness. I find it easier to add the styles directly inline rather than in a CSS file when I'm in the process of designing something new. I usually don't have the final picture in mind and I go back and forth a lot between different ideas. It's more comfortable to go through this design phase when the code is in one place and doesn't need to be interrupted by a build.

With https://github.com/Tatoeba/tatoeba2/commit/645b58ec2e29311b17da1c930f29855119049347 it is a bit less of an issue.